src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
```

Comment:
The `calc` function is used correctly here to center the element, but according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid complex calculations in CSS when simpler solutions are available. If the width of the hand is a known fixed size, consider using a simpler method like `transform: translateX(-50%)` combined with `left: 50%` to achieve the same effect without `calc`.